src/isomorphic/modern/class/ReactBaseClasses.js

```diff
@@ -21,7 +21,28 @@ var warning = require('fbjs/lib/warning');
 /**
  * Base class helpers for the updating state of a component.
  */
+
+const addUpdaterMutationWarn = context => {
```

Comment:
As long as you only access `addUpdaterMutationWarn` inside a `__DEV__` block it should work fine. For example, in `ReactFiberClassComponent` you can see that [`startPhaseTimer`](https://github.com/facebook/react/blob/b1b4a2fb252f26fe10d29ba60d85ff89a85ff3ec/src/renderers/shared/fiber/ReactFiberClassComponent.js#L43-L58) is defined in a `__DEV__` block and only accessed in other `__DEV__` blocks.